tests/modules/blockedclient.c

```diff
@@ -8,6 +8,7 @@
 #include <stdio.h>
 #include <pthread.h>
 #include <strings.h>
+#include "sds.h"
```

Comment:
where does this sds.h come from? we can't use redis internals.

Reply:
The runtime linker links us to the sds in Redis. But you are right, will change to avoid using sds.